packages/core/contracts/oracle/implementation/Voting.sol

```diff
@@ -255,6 +256,14 @@ contract Voting is
         require(time <= blockTime, "Can only request in past");
         require(_getIdentifierWhitelist().isIdentifierSupported(identifier), "Unsupported identifier request");
 
+        // As of December 2020, the current Ethereum gas limit is 12.5 million. This function's gas primarily
+        // comes from computing a Keccak-256 hash in _encodePriceRequest and writing a new PriceRequest to
+        // storage. We have empirically determined an ancillary data limit of 8192 bytes that keeps this function
+        // well within the gas limit at ~8 million gas. To learn more about the gas limit and EVM opcode costs go here:
+        // - https://etherscan.io/chart/gaslimit
+        // - https://github.com/djrtwo/evm-opcode-gas-costs
+        require(ancillaryData.length <= 8192, "Invalid ancillary data");
```

Comment:
we might want to expose somewhere the limit as a public constant so that this check can be done before tx is sent

Reply:
Good idea

Reply:
Done